generate/template/astronomy.kt

```diff
@@ -1989,7 +1994,7 @@ internal fun peakMoonShadow(searchCenterTime: Time): ShadowInfo {
     val window = 0.03       // days before/after new moon to search for minimum shadow distance
     val t1 = searchCenterTime.addDays(-window)
     val t2 = searchCenterTime.addDays(+window)
-    val tx = search(moonShadowSlopeContext, t1, t2, 1.0) ?:
+    val tx = search(t1, t2, 1.0, moonShadowSlopeContext) ?:
```

Comment:
This should've be done in the previous PR, not sure how I've missed it...

Reply:
OK, I just pushed this fix already. I figured we were both changing things at the same time?

Reply:
True, and great that git didn't need a rebase! :)